community.md

```diff
@@ -72,6 +72,16 @@ Some quick tips when using email:
 and include only a few lines of the pertinent code / log within the email.
 - No jobs, sales, or solicitation is permitted on the Apache Spark mailing lists.
 
+<a name="chat"></a>
+<h4>Chat Rooms</h4>
+
+Chat rooms are great for quick questions or discussions on specialized topics. The following chat rooms are not officially part of Apache Spark; they are provided for reference only.
+<ul>
+  <li>
+    <a href="https://gitter.im/spark-scala/Lobby">Spark with Scala</a> is for questions and discussions related to using Spark with the Scala programming language.
```

Comment:
> no idea what the theory is about HTML vs markdown but this can use simple -

agree, I somehow missed that some parts of the document actually used markdown (html drowns the content in tags), hence to stay consistent with the rest I used html.

Should I change this or would you like to do it in the clean up you mention?